notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<blockquote>Use coordinates to plot a spectrum of a known member in a dwarf galaxy. </blockquote><p>Please comment out this because there's no spectum to show. But do not delete the line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>